### PR TITLE
chore(api): build api clients using a prebuilt image

### PR DESCRIPTION
**Pull request by Bobgy**
**Description of your changes:**
Follow up of https://github.com/kubeflow/pipelines/pull/4906#issuecomment-778945609
/assign @capri-xiyue @NikeNano 

**Checklist:**
- [ ] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention). 

- [ ] Do you want this pull request (PR) cherry-picked into the current release branch?

    [Learn more about cherry-picking updates into the release branch](https://github.com/kubeflow/pipelines/blob/master/RELEASE.md#cherry-picking-pull-requests-to-release-branch).

### CODE REVIEW

File: backend/api/Makefile [hunk 1]
```diff
@@ -1,15 +1,36 @@
+# Makefile to generate KFP api clients from proto.
+
 IMAGE_TAG=kfp-api-generator
+# Contact one of KFP maintainers if this remote image needs an update.
```
Comment (NikeNano) [inline backend/api/Makefile:4]: Should we refer to where the owner can be found?

**Review by NikeNano — APPROVED** [review]
nit: we could add an reference to where the owners/maintainers could be found. 
/LGMT

File: backend/api/Makefile (hunk 1 above)
Comment (Bobgy) [reply backend/api/Makefile]: Good point!

**Pull request merged**